api/tests/unit/infrastructure/serializers/jsonapi/certification-result-information-serializer_test.js

```diff
@@ -2,18 +2,20 @@ const { expect, domainBuilder } = require('../../../../test-helper');
 const serializer = require('../../../../../lib/infrastructure/serializers/jsonapi/certification-result-information-serializer');
 const CertificationResultInformation = require('../../../../../lib/domain/read-models/CertificationResultInformation');
 const Assessment = require('../../../../../lib/domain/models/Assessment');
-const cleaCertificationStatusRepository = require('../../../../../lib/infrastructure/repositories/clea-certification-status-repository');
 
-describe('Unit | Serializer | JSONAPI | certification-result-information-serializer', function() {
+describe('Unit | Serializer | JSONAPI | certification-result-information-serializer', () => {
```

Comment:
**Nitpick:** Pour les tests, les arrow functions peuvent poser problême. Pas certain qu'il faille faire ce refacto

Reply:
Je crois que ça pose essentiellement problème sur les tests front, car on perd le contexte du rendering de test Ember je crois (et du this.owner notamment). Il me semble que côté api y'a pas de problème, me trompe-je ?